src/proxy.ts

```diff
@@ -70,6 +75,28 @@ export function createProxyEndpoints(app: Application, options: Options) {
     await fetchAndRespond('GET', req.path, req.query, req.headers, req.body, res, options)
   })
   app.post(POST_PROXY_ENDPOINTS, async (req, res) => {
+    if (options.filteringActive && req.path.startsWith('/bytes') && req.method === 'POST') {
```

Comment:
don't we want to enable filtering for GET ? I mean, if the user enables it then do we want to display everything ?

Reply:
If you mean filtering out the content that was flagged, that should happen on the client side, because the user will be able to switch the filter on or off. So the client should get all the content as it is, and do the filtering on the UI.

Reply:
@bosi95 Or do you think it is easier if the client can send it as an option to the gateway request and the gateway does the filtering? Since most of the related logic is on the client side, I would prefer to keep this one there as well, but if that is difficult, we can try to have it on the gateway.